### ISSUE TITLE
Installation script fails when (trying to) self-host the panel

### ISSUE THREAD

**Issue by diamkil** [lucko/LuckPermsWeb#296]
It seems there's an error regarding Yorkie in NPM when trying to run the script (does the same error when doing 'npm install yorkie') https://bytebin.lucko.me/cL7ym77OpV

**Comment by thetayloredman** [comment]
Why are you using the panel yourself tho

**Comment by diamkil** [comment]
> Why are you using the panel yourself tho

I like to self-host my stuff under our domain and why not it's there for a reason

**Comment by BrainStone** [comment]
@diamkil hi there. I'm the author of the installer. I was busy this past week, sorry for that.  
Sadly the paste link has expired. Can you get me another one?

**Comment by diamkil** [comment]
> @diamkil hi there. I'm the author of the installer. I was busy this past week, sorry for that.  
> 
> Sadly the paste link has expired. Can you get me another one?

Yeah I think the fix is simply to run the npm install command in the script with the -unsafe-perm (I think that's the flag) because it fails when running the script as root, I'll try to get the log for you

**Comment by BrainStone** [comment]
That would be apprechiated.  
I've mainly tested it as root so that would be surprising to me...
